### ISSUE TITLE
installing compdb via pip does not install configparser dependency.

### ISSUE THREAD

**Issue by rasjani** [Sarcasm/compdb#1]
Console log looks something like this:

```
C:\src\work\apps>pip install compdb
Collecting compdb
  Downloading compdb-0.0.2-py2.py3-none-any.whl
Installing collected packages: compdb
Successfully installed compdb-0.0.2


C:\src\work\apps>compdb
Traceback (most recent call last):
  File "c:\python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27\Scripts\compdb.exe\__main__.py", line 5, in <module>
  File "c:\python27\lib\site-packages\compdb\cli.py", line 9, in <module>
    import compdb.config
  File "c:\python27\lib\site-packages\compdb\config.py", line 3, in <module>
    import configparser
ImportError: No module named configparser
```

**Comment by Sarcasm** [comment]
Congratulation on opening the first issue.
I have identified the issue I think.
For the releases I have created a setup.cfg which generates [Universal Wheels](https://packaging.python.org/distributing/#universal-wheels):

* https://github.com/Sarcasm/compdb/blob/829a1ff05058933b9613d3b96046b74db57f9cac/setup.cfg

As it happens, the wheel is actually not universal, this dependence to `configparser` probably requires to make 2 distributions, one for Python 2 and one for Python 3.

I will generate a new version with a fix, in the meantime `configparser` will have to be installed manually for python2 installs.
